src/client/src/ui/analytics/components/LastPosition.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react'
+import numeral from 'numeral'
+import { LastPositionWrapper, LastPositionStyle, USDspan } from './styled'
+
+interface LastPositionProps {
+  lastPos?: number | undefined
+}
+
+interface FormattedLastPosition {
+  lastPosition: string | undefined
+  color: string
+}
+
+const formatLastPosition: (lastPos: number) => FormattedLastPosition = (lastPos: number) => {
```

Comment:
The naming here isn't optimal!  The function takes a number and returns a type named `FormattedLastPosition` - but that type represents something to do with `color` also and not just a formatted version of the number that was provided.  I'd rethink the type and method name.